src/components/Editor/customKeymap.ts

```diff
@@ -22,27 +23,31 @@ export default Prec.highest(
     },
     {
       key: 'Ctrl-b',
+      mac: 'Mod-b',
```

Comment:
Might be a dumb question but can't we just use `mod` key bindings for everything rather than specifically for mac mode?
Read more about what mod keys are here - https://unix.stackexchange.com/questions/119212/understanding-window-manager-terminology-mod-keys-meta-keys-and-key-naming-co

Reply:
That is actually a good point, I re-read the documentation https://codemirror.net/docs/ref/#view.KeyBinding. I think it should work, let me try and check it here...